src/world.rs

```diff
@@ -102,6 +131,21 @@ impl World {
         &self.archetypes
     }
 
+    /// Insert components for a given [Entity](struct.Entity.html).
+    ///
+    /// # Panics
+    ///
+    /// Panics if there is already one of the component types for the entity.
```

Comment:
I stumbled over this while reading. Maybe "if one of the components is already present for the entity"?